spec/spec_helper_acceptance.rb

```diff
@@ -72,7 +68,12 @@ def parser_opts
   {
     :main => {:stringify_facts => false, :parser => 'future', :color => 'ansi'},
     :agent => {:stringify_facts => false, :cfacter => true, :ssldir => '$vardir/ssl'},
-    :master => {:stringify_facts => false, :cfacter => true},
+  }
+end
+
+def server_opts
+  {
+    :master => {:autosign => true},
```

Comment:
it doesn't look like `server_opts` isn't used anymore (previously we called `master_opts`)